### PR TITLE
Added "Logout" to profile page

### PR DESCRIPTION
**Pull request by localastronaut**
Not a commonly used item. Made more sense to have it on the profile.
Need to find a good place for it that is still easy to access, but
don’t see users needing to click it as often as they would other links.

### CODE REVIEW

**Comment by RevolutionTech** [comment]
Honestly, I don't really agree with this change. I think it's pretty typical for sites to have Logout as an option in the nav bar, or within a dropdown in the upper-right of the nav bar.

Maybe a better solution would be to replace the Profile button with an avatar. When you hover over the avatar there's a profile button and a logout button, though clicking the avatar could also take you to profile page. Then we could place the user's Facebook profile photo / Google photo if they sign up via one of those options. Later we could let users upload their own photo too.

**Comment by localastronaut** [comment]
That would be amazing. I thought it would be better to have a drop down down on the upper-right nav and have logout be one of them, but I didn't know how to do that so I just put it in the profile in the mean time. 

The main challenge I was running into with the logout was using it on mobile it caused the whole nav bar to bump down and since it wasn't something that was used regularly as artists or home or profile I thought it would be good to not have it take up so much viewing space. 

Still want to do some work on the nav bar. I think it might be cool to have one of those hamburger menus when viewing it on small devices do we don't run into the problem of the nav bar taking up so much space.

**Comment by RevolutionTech** [comment]
There's some helpful examples in the Zurb Foundation docs that essentially show how to do this, if you want to check it out.
http://foundation.zurb.com/sites/docs/top-bar.html

Foundation supports hamburger menus too. Check out this page:
http://foundation.zurb.com/sites/docs/off-canvas.html

File: templates/base.html [hunk 1]
```diff
@@ -19,18 +19,43 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
+    <div class="off-canvas-wrapper">
```
Comment (RevolutionTech) [inline templates/base.html]: I counted the `<div>` and I noticed you added 5 new open ones, but only closed 4 so one of these are off.

**Comment by RevolutionTech** [comment]
As discussed, I'm deleting this branch as `nav-bar-formatting` has replaced it.

**Pull request closed without merging**